api/README.md

```diff
@@ -16,3 +16,8 @@ Execute tests on the command line or through IntelliJ
 To run a specific test alone, add `--test` + the \
 [fully qualified class name](https://docs.gradle.org/current/userguide/java_testing.html#full_qualified_name_pattern)\
 to the command. For example, the command for running AppTest.kt would be: `./api/gradlew -p api/app :app:test --tests AppTest`
+
+## Database migration
+Database uses [flyway](https://flywaydb.org/documentation/database/h2) to run migration, the scripts are located in \
+`api/resources/db/migration`. Follow the naming pattern in migration folder to alter the schema \
```

Comment:
This is a problem for adding the dummy data - the schema needs to exist before the data is imported. I suggest you back this out for now and stick with the simpler approach of just running the schema import file (no Flyway). We did say we don't need to worry about migrations for now, since we can just destroy and rebuild the db every time we deploy.